hack/update-rhcos-bootimage.py

```diff
@@ -8,12 +8,13 @@
 # builds.  Do not try to e.g. point to RHT-internal endpoints.
 RHCOS_RELEASES_APP = 'https://releases-art-rhcos.svc.ci.openshift.org'
 
-dn = os.path.abspath(os.path.dirname(sys.argv[0]))
-
 parser = argparse.ArgumentParser()
 parser.add_argument("meta", action='store')
+parser.add_argument("arch", action='store', choices=['amd64'])
 args = parser.parse_args()
 
+metadata_dir = os.path.join(os.path.dirname(sys.argv[0]), f"../data/data")
```

Comment:
nit: no need for the [f-string][1] here, since there are no expansions.

[1]: https://www.python.org/dev/peps/pep-0498/